packages/acceptance-tests/pkg-tests-specs/sources/commands/up.test.ts

```diff
@@ -33,13 +31,47 @@ describe(`Commands`, () => {
       }, async ({path, run, source}) => {
         await run(`up`, `@types/*@1.0.0`);
 
-        await expect(readManifest(path)).resolves.toStrictEqual({
+        await expect(xfs.readJsonPromise(ppath.join(path, Filename.manifest))).resolves.toStrictEqual({
           dependencies: {
             [`@types/is-number`]: `1.0.0`,
             [`@types/no-deps`]: `1.0.0`,
           },
         });
       }),
     );
+
+    test(
+      `it should add a new regular dependency to the current project (resolved tag)`,
```

Comment:
I don't think this describes what it's testing as it doesn't add a new dependency.